tensilelite/Tensile/TensileInstructions/Enums.py

```diff
@@ -93,6 +93,7 @@ class CvtType(Enum):
     CVT_PK_FP8_to_F32 =  9
     CVT_PK_BF8_to_F32 = 10
     CVT_PK_F32_to_FP8 = 11
+    CVT_PK_F32_to_BF8 = 12
 
```

Comment:
Weird, this has been added in #564 and should be existing in the latest develop branch...maybe you haven't done rebase ?